### PR TITLE
[17.0][MIG] account_financial_report_operating_unit : Migration to 17.0

### PR DESCRIPTION
**Pull request by traitrantrobz**
Depends on:

- [x] https://github.com/OCA/account-financial-reporting/pull/1101

- [x] https://github.com/OCA/operating-unit/pull/670

### CODE REVIEW

**Review by leemannd — APPROVED** [review]
Code looks good. There are no unit tests for the moment

**Comment by yankinmax** [comment]
@traitrantrobz can you pls remove dependency commit and rebase?
- [ ] https://github.com/OCA/operating-unit/pull/670 is now merged

**Review by yankinmax — APPROVED** [review]

File: account_financial_report_operating_unit/report/open_items.py [hunk 1]
```diff
@@ -13,10 +13,10 @@ def _get_report_values(self, docids, data):
 
     @api.model
     def _get_move_lines_domain_not_reconciled(
-        self, company_id, account_ids, partner_ids, target_move, date_from
+        self, company_id, account_ids, partner_ids, only_posted_moves, date_from
```
Comment (yankinmax) [inline account_financial_report_operating_unit/report/open_items.py:16]: Is this change needed?

**Review by yankinmax — APPROVED** [review]

File: account_financial_report_operating_unit/report/open_items.py (hunk 1 above)
Comment (traitrantrobz) [reply account_financial_report_operating_unit/report/open_items.py:16]: @yankinmax I want to synchronize the parameter names of the original function.
